### ISSUE TITLE
Empty Annotation window

### ISSUE THREAD

**Issue by spujol** [Slicer/Slicer#5200]
An Annotation window appears when clicking on the screenshot icon in the top banner. 

An empty Annotation window remains after clicking on OK to create the screenshot.

Slicer4.110.0-2020-09-10 on MacOSX 10.14.16

<img width="1680" alt="Screen Shot 2020-09-23 at 2 36 59 PM" src="https://user-images.githubusercontent.com/1847492/94013632-a56eef80-fdaa-11ea-8620-b8957a08ca77.png">
<img width="1680" alt="Screen Shot 2020-09-23 at 2 35 58 PM" src="https://user-images.githubusercontent.com/1847492/94013646-a869e000-fdaa-11ea-8b10-6c2ebbff2071.png">

**Comment by pieper** [comment]
Thanks for reporting this, I've seen it too; I think it's only on mac.  The workaround is to close the second empty window.

**Comment by jamesobutler** [comment]
> I think it's only on mac.

Tested on Windows and it does not happen so you're probably right.

**Comment by lassoan** [comment]
We need to rethink this feature as we plan to retire Annotations module and the feature offers very little that is not covered by other modules. Scale factor would be unique feature but it does not work (at least on my computer): when it actually renders in higher resolution in 3D (and not just resize the current image) then the output image gets garbled.

- You write screenshots to write to file using Screen capture module.
- Scene views capture screenshots and saves them into the scene (along with description and the entire scene).
- SlicerMorph Animator can record screenshot (videos), too.

What do you think about replacing this features with a similar popup window, implemented in Python, using ScreenCapture module logic, that would offer saving of current view layout directly to file? If we can make scale factor work correctly then we add it, too.

**Comment by pieper** [comment]
Yes, scale factor is a very nice feature if we can find a way to make it work.  From time to time people want to make poster-sized renderings.  It used to work well but I think now it's broken at the VTK level in various ways (I think GPU volume rendering is broken but there may be other things.

Personally, I never use the slicer annotations screen capture, but always rely on the OS tools, so I wouldn't mind seeing this feature go away.

**Comment by jamesobutler** [comment]
We use some screen capture tools within Slicer and would be fine if it was reimplemented in Python using ScreenCapture module logic.  We generally use it instead of OS tools to automatically generate images of volumes with their segmentation to save to disck and then for generating automated reports for longitudinal study analysis.
